packages/insomnia-app/app/ui/components/modals/settings-modal.js

```diff
@@ -65,11 +65,36 @@ class SettingsModal extends PureComponent {
     this.modal.hide();
   }
 
-  async _handleChangeTheme(theme, persist = true) {
-    await setTheme(theme);
+  async _handleChangeTheme(themeName, colorScheme, persist = true) {
+    const { settings } = this.props;
+
+    let patch;
+    switch (colorScheme) {
+      case 'light':
+        patch = { theme: themeName };
```

Comment:
I think this should be setting `lightTheme`?

```suggestion
        patch = { lightTheme: themeName };
```